ChiModules/LinearBoltzmannSolver/IterativeOperations/lbs_setsource.cc

```diff
@@ -77,15 +79,15 @@ SetSource(LBSGroupset& groupset,
       {
         unsigned int ell = m_to_ell_em_map[m].ell;
 
-        size_t ir = full_cell_view.MapDOF(i, m, 0);
-
+        size_t uk_map = full_cell_view.MapDOF(i, m, 0); //unknown map
 
-        //=================================== Loop over groupset groups
+        //============================== Loop over groupset groups
         for (size_t g = gs_i; g <= gs_f; ++g)
         {
-          //=================================== Apply material source
-          if (apply_mat_src && (ell == 0))
-            destination_q[ir + g] += src[g];
+          if ( apply_mat_src and (ell == 0) and (not options.use_src_moments))
```

Comment:
The paren usage here makes this a bit confusing to read.  Could we simplify it a bit?

Reply:
The process of simplification would then be subjected to subjective evaluation. (that sounds like something from the matrix haha).
Could you tell me what you would like to see?

Reply:
How about this?
```c++
if (not options.use_src_moments) //using regular material src
{
  if (apply_mat_src and ell == 0)
    destination_q[uk_map + g] += src[g];
}
else if (apply_mat_src) //using ext_src_moments
  destination_q[uk_map + g] += ext_src_moments_local[uk_map + g];
```